src/react-app-env.d.ts

```diff
@@ -1 +1,29 @@
 /// <reference types="react-scripts" />
+
+interface User {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+  address: {
+    street: string;
+    suite: string;
+    city: string;
+    zipcode: string;
+    geo: {
+      lat: string;
+      lng: string;
+    }
+  }
+}
```

Comment:
You don't need to describe all properties. You should describe only what you are going to use.